example/macos/Podfile

```diff
@@ -2,6 +2,12 @@ require_relative '../../node_modules/react-native-test-app/macos/test_app.rb'
 
 workspace 'AsyncStorageExample.xcworkspace'
 
+options = {
+  :fabric_enabled => true,
+  :hermes_enabled => false,
+  :turbomodule_enabled => true,
+}
+
```

Comment:
The podspec requires the `RCT_NEW_ARCH_ENABLED` be passed to `pod install`. Enabling these options w/o the flag will not enable new arch.

https://github.com/j-piasecki/react-native-async-storage/blob/8a47c53c974c666419a290ff94b6f3548e797c5e/RNCAsyncStorage.podspec#L5

Reply:
Are you sure about this? I've just tested in `master` and it seems to be working?

```diff
diff --git a/RNCAsyncStorage.podspec b/RNCAsyncStorage.podspec
index 5d74402..0552c43 100644
--- a/RNCAsyncStorage.podspec
+++ b/RNCAsyncStorage.podspec
@@ -2,6 +2,10 @@ require 'json'

 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))

+p '%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%'
+p ENV['RCT_NEW_ARCH_ENABLED']
+p '%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%'
+
 Pod::Spec.new do |s|
   s.name         = "RNCAsyncStorage"
   s.version      = package['version']
```

Output:

```
% RCT_NEW_ARCH_ENABLED=1 pod install
...
Auto-linking React Native module for target `ReactTestApp`: ReactTestApp-DevSupport
Analyzing dependencies
"%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%"
"1"
"%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%"
Downloading dependencies
Generating Pods project
Setting REACT_NATIVE build settings
Setting CLANG_CXX_LANGUAGE_STANDARD to c++17 on /Users/tido/Source/async-storage/node_modules/.generated/macos/ReactTestApp.xcodeproj
Pod install took 11 [s] to run
```

Reply:
I've just cleaned out my clone in case I had something stale. Still getting the same results.